README.md

```diff
@@ -221,6 +221,49 @@ Write code that is reusable, scalable and testable.
 
 See https://en.wikipedia.org/wiki/Magic_number_(programming)
 
+
+#### Default in a switch
+
+Every `switch` must have a `default`. If there is no need to handle the `default`, either throw an
+Error or add a comment that the default is explicit ignored.
```

Comment:
- ... throw an \`Error\` / (or just error)
- ... that the default is explicitly ignored.